app/src/js/utils/table-config/granules.js

```diff
@@ -275,7 +275,7 @@ const containsPublishedGranules = (selectedGranules) => {
   let publishedGranules = [];
 
   if (Array.isArray(selectedGranules) && selectedGranules.length > 0) {
-    publishedGranules = selectedGranules.filter((g) => g.published === true);
+    publishedGranules = selectedGranules.filter((g) => g && g.published === true);
```

Comment:
The problem was showing up here but this stuff is now redundant, but will prevent us from seeing it in the future.